src/test/java/com/google/api/codegen/testdata/java_smoke_test_library.baseline

```diff
@@ -0,0 +1,48 @@
+============== file: src/test/java/com/google/gcloud/pubsub/spi/LibraryServiceSmokeTest.java ==============
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.gcloud.pubsub.spi;
+
+import autovalue.shaded.org.apache.commons.lang.builder.ReflectionToStringBuilder;
```

Comment:
Done. We need to added the org.apache.commons.lang dependency into the pom file.